src/plugins/data/common/exports/export_csv.tsx

```diff
@@ -21,39 +21,51 @@ interface CSVOptions {
   escapeFormulaValues: boolean;
   formatFactory: FormatFactory;
   raw?: boolean;
+  sortedColumns?: string[];
 }
 
 export function datatableToCSV(
   { columns, rows }: Datatable,
-  { csvSeparator, quoteValues, formatFactory, raw, escapeFormulaValues }: CSVOptions
+  { csvSeparator, quoteValues, formatFactory, raw, escapeFormulaValues, sortedColumns }: CSVOptions
```

Comment:
To differentiate `sortedColumns` from `columns` (because it was confusing to read at first), we could rename it to `columnsSorting`